plugins/Toolbox/src/Toolbox.py

```diff
@@ -61,13 +62,15 @@ def __init__(self, application: CuraApplication) -> None:
             "authors":             [],
             "packages":            [],
             "updates":             [],
+            "installed_packages":   [],
         }  # type: Dict[str, List[Any]]
 
         # Models:
         self._models = {
             "authors":             AuthorsModel(self),
             "packages":            PackagesModel(self),
             "updates":             PackagesModel(self),
+            "installed_packages":  PackagesModel(self),
```

Comment:
I think it's better to rename this to prevent confusion. This doesn't hold the installed packages, but the packages the user is subscribed to.

Reply:
I'm very much open to name-suggestions!
I've went for the "installed_packages" because locally (in Cura) they are shown as "Installed", but on the Cloud as "Subscribed" to. Anyway, I don't mind renaming them to something more specific, just give me a name.

1. "subscribed_packages" ?
2. "subscribed" ?
3. ...

Reply:
I think subscribed_packages would fit the bill best.